### ISSUE TITLE
Bug accessing the dedicated metadata url

### ISSUE THREAD

**Issue by edmund-dunn** [simplesamlphp/simplesamlphp#1842]
# **Help us to keep this issue-tracker clean!  For questions or support, please refer to our [mailing lists](https://simplesamlphp.org/lists)**

**Describe the bug**
I just installed v2.1 for testing on our drupal 10 site. I copied the update config.php.dist and moved our settings over into it. I can access the metadata at `/simplesaml/module.php/admin/federation` but not at `/simplesaml/module.php/saml/sp/metadata/default-sp` When I try to access the dedicated URL I get an error.

```
SimpleSAML\Error\Error: UNHANDLEDEXCEPTION
Backtrace:
2 public/_include.php:31 (SimpleSAML_exception_handler)
1 /var/www/cms/docroot/vendor/symfony/error-handler/ErrorHandler.php:537 (Symfony\Component\ErrorHandler\ErrorHandler::handleException)
0 [builtin] (N/A)
Caused by: TypeError: SimpleSAML\HTTP\RunnableResponse::sendContent(): Return value must be of type SimpleSAML\HTTP\RunnableResponse, null returned
Backtrace:
2 src/SimpleSAML/HTTP/RunnableResponse.php:68 (SimpleSAML\HTTP\RunnableResponse::sendContent)
1 /var/www/cms/docroot/vendor/symfony/http-foundation/Response.php:423 (Symfony\Component\HttpFoundation\Response::send)
0 public/module.php:14 (N/A)
```

**To Reproduce**
1. Go to `/simplesaml/module.php/saml/sp/metadata/default-sp`
2. See error

**Expected behavior**

We should get the metadata XML file.

**Screenshots or logs**
If applicable, add screenshots and/or log files to help explain your problem.
![CleanShot2023-08-10at12 18 06](https://github.com/simplesamlphp/simplesamlphp/assets/109987005/011f9f52-e860-44b7-844a-3865be96e472)

**Additional context**

**Comment by tvdijen** [comment]
@edmund-dunn Would you mind trying again?

**Comment by tvdijen** [comment]
Closed in adac8e87ac2d346fa0f309b841735f8d525a7df1
